### PR TITLE
LAKSHYA_VERMA: JITTER CLICK GAME

### PR DESCRIPTION
**Pull request by lakshyeahh**

### CODE REVIEW

File: src/assets/games/jitter/App.css
Comment (ken1000minus7) [inline src/assets/games/jitter/App.css:1]: Looks great! But what is the use of this file? It's not imported anywhere. Remove this file if not required

**Comment by Ishwarendra** [comment]
@lakshyeahh fix the conflict and push again. Also make sure to delete un-needed files.

File: src/assets/games/jitter/App.css
Comment (lakshyeahh) [reply src/assets/games/jitter/App.css:1]: Done

**Review by Ishwarendra — APPROVED** [review]
lgtm.

**Pull request merged**